### ISSUE TITLE
Default menu items disappears from sketchybar

### ISSUE THREAD

**Issue by 9tin9tin9** [FelixKratz/SketchyBar#341]
Default menu items sometimes disappears from sketchybar. Although I notice it mostly after switching spaces, I believe its not the only time when the items disappears.

**Comment by FelixKratz** [comment]
I have noticed this too. It has something to do with the animations in macOS. When I disable those animations (with yabai) the aliases do not disappear anymore.

I think it could be that I have to check if the display is animating before capturing the surface of the default bar item.

**Comment by FelixKratz** [comment]
Hmm, so it seems that calling
```c
extern void SLSCaptureWindowsContentsToRectWithOptions(uint32_t cid, uint64_t* wid, uint32_t count, CGRect bounds, uint32_t flags, CGImageRef* image);
```
during a system animation leaves the macOS bar windows in a state where their surfaces can not be captured again until another event updates them (e.g. another space change or revealing the macOS bar).

**Comment by 9tin9tin9** [comment]
I've set `alias.update_freq=1` and the items doesn't seem to disappear (or at least I haven't notice it). Sketchybar doesn't seem to use much resources because of this so I will just stick with this solution. Thank you for your response :)

**Comment by FelixKratz** [comment]
I think `alias.update_freq=5` will reduce the probability of this happening on space change, while `alias.update_freq=1` is default. I will have to look for a way to detect these animations…

**Comment by FelixKratz** [comment]
I think this commit should fix it: 2f01d41
